website/blog/2024-07-11-what-is-a-data-lakehouse.md

```diff
@@ -0,0 +1,154 @@
+
+## Real World Examples
+ByteDance, Notion and Halodoc are some of the examples of how lakehouse architecture is being adopted in the industry. [ByteDance](https://hudi.apache.org/blog/2021/09/01/building-eb-level-data-lake-using-hudi-at-bytedance/) has built an exabyte-level data lakehouse using Apache Hudi to enhance their recommendation systems. The implementation of Hudi's Merge-on-read (MOR) tables, indexing, and Multi-Version Concurrency Control (MVCC) features allow ByteDance to provide real-time machine learning capabilities, providing instant and relevant recommendations.
+
+[Notion](https://www.notion.so/blog/building-and-scaling-notions-data-lake) scaled its data infrastructure by building an in-house lakehouse to handle rapid data growth and meet product demands, especially for Notion AI. The architecture uses S3 for storage, Kafka and Debezium for data ingestion, and Apache Hudi for efficient data management. This setup resulted in significant cost savings, faster data ingestion, and enhanced capabilities for analytics and product development.
+
+Similarly, [Halodoc's](https://blogs.halodoc.io/lake-house-architecture-halodoc-data-platform-2-0/) adoption of a lakehouse architecture allows them to enhance healthcare services by enabling real-time processing and analytics. This architecture helps Halodoc tackle challenges associated with managing vast healthcare data volumes, thus improving patient care through faster, more accurate decision-making and supporting both batch and stream processing crucial for timely health interventions.
+
+## Key Data Lakehouse Technologies
+
+### Open Source Solutions
+
+#### Apache Hudi
+Apache Hudi is an open source [transactional data lakehouse platform](https://hudi.apache.org/docs/hudi_stack) built around a database kernel. It provides table-level abstractions over open file formats like Apache Parquet and ORC thereby delivering core warehouse and database functionalities directly in the data lake and supporting transactional capabilities such as updates and deletes. 
+
+Hudi also incorporates critical table services tightly integrated with its database kernel. These services can be run automatically, managing aspects like table bookkeeping, metadata, and storage layouts across both ingested and derived data. These capabilities, combined with specific platform services (ingestion, catalog sync tool, admin CLI, etc.) in Hudi, elevates its role from merely a table format to a comprehensive and robust data lakehouse platform. Apache Hudi has a broad support for various data sources and query engines, such as Apache Spark, Apache Flink, AWS Athena, Presto, Trino and StarRocks.
+
+![Hudi Stack](/assets/images/blog/hudistack/hstck_new.png)
```

Comment:
is this a new image or an existing one ?

Reply:
this is an existing image of the Hudi Stack referenced here.